modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -5,61 +5,45 @@
 [id="odc-creating-a-binding-connection-between-components_{context}"]
 = Creating a binding connection between components
 
-You can establish a binding connection with Operator-backed components.
+You can create a binding connection between components, as demonstrated in the following example that uses a PostgresSQL database service and a Spring PetClinic sample application.
```

Comment:
@Dhruv-Soni11 Please leave a space after L3. Refer to [Module file metadata](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)